### ISSUE TITLE
Deprecation warning on normalize_whitespace with capybara >= 3.0

### ISSUE THREAD

**Issue by trusche** [thoughtbot/capybara-webkit#1065]
Hi!

Since [this commit](https://github.com/teamcapybara/capybara/commit/b0e82b092531b2fb7def85e4e6350a2493097fcd#diff-1267c558eb1eedb953ffd99b2ed187ec) on Capybara, first released with capybara  3.0.0.rc2, this deprecation warning is thrown:

> DEPRECATED: Capybara::Helpers::normalize_whitespace is deprecated, please update your driver

This seems to be fixed on master in [this commit](https://github.com/thoughtbot/capybara-webkit/commit/778ec7f43cd1e8de272c2fc7023ce9d52fa74f9d#diff-6377fd92e58358ecd94b575d89c6f709). Any chance of a new release?

Thank you!

**Comment by seanfcarroll** [comment]
Had the same issue during a Rails 5.2 upgrade, tracking master removed the warnings.

**Comment by sbilharz** [comment]
These warnings come in hundreds which is quite annoying. What's wrong with a patch release?

**Comment by giedriusr** [comment]
Any news on this?

**Comment by Hirurg103** [comment]
Switching to the branch `master` should fix the issue until the next version of `capybara-webkit` will be released

```ruby
# Gemfile
group :test do
  gem 'capybara-webkit', github: 'thoughtbot/capybara-webkit', branch: 'master'
end
```

**Comment by derosm2** [comment]
Any time frame on the next release? Much appreciated, thanks!

**Comment by twalpole** [comment]
@Hirurg103 Those errors are because you've updated to Capybara 3 and the master branch  of capybara-webkit properly supports Capybara 3, whereas 1.15.0 does not.  The details are in Capybaras upgrade guide - https://github.com/teamcapybara/capybara/blob/master/UPGRADING.md#node.  Capybara 3.5.0 also adds a `normalize_ws` option to the text matchers -https://github.com/teamcapybara/capybara/blob/3.5_stable/History.md#added - to mimic Capybara 2 behavior if you dont want/need to actually verify what's displayed

**Comment by Hirurg103** [comment]
@twalpole with capybara 3.5.0 and capybara-webkit 1.15.0 I had the build green. But after switching to capybara-webkit master branch and leaving capybara version the same it started to fail with the spacing error above

**Comment by twalpole** [comment]
@Hirurg103 Yes. As I stated in my comment capybara-webkit 1.5.0 does not meet the Capybara 3 requirements for returning the text of an element, whereas the master branch does.  The errors you get about text matching when using the master branch are correct for Capybara 3 and would be returned if you use any of the other drivers which have released Capybara 3 compliant versions.

**Comment by Hirurg103** [comment]
@twalpole Before the upgrade I used capybara `2.13.0` and capybara-webkit `1.14.0` and the build was green. After the upgrade to capybara `3.5.0` and capybara-webkit `1.15.0` the build was still green. But after switching to the master branch of capybara-webkit it started to fail

**Comment by twalpole** [comment]
@Hirurg103 The breaking changes are from Capybara 2.x to Capybara 3.x as mentioned in the upgrade.md I linked previously - you just didn’t see them until using `capybara-webkit’ master branch because `capybara-webkit` 1.15 doesn’t fully support Capybara 3.  I don’t know how else to explain it to you. With Capybara 3 your tests SHOULD be failing.

**Comment by Hirurg103** [comment]
@twalpole thank you for the explanation! Now I see why my build still passing after upgrade to capybara 3 and capybara webkit 1.15.

**Comment by dougjohnston** [comment]
Here's the fix that worked for me (which @twalpole already mentioned above):

**Before**
`expect(page).to have_content(message)`

**After**
`expect(page).to have_content(message, normalize_ws: true)`

From my understanding, the `normalize_ws` flag just forces Capybara to fallback to the old way of comparing a string. [Documentation](https://www.rubydoc.info/gems/capybara/Capybara%2FNode%2FMatchers%3Ahas_text%3F)

**Comment by Hirurg103** [comment]
@twalpole thanks a lot for the patience 😅 , I haven't read the Capybara upgrade guide for the first time properly

**Comment by krisleech** [comment]
For now, either:

```ruby
gem "capybara"
gem 'capybara-webkit', git: 'https://github.com/thoughtbot/capybara-webkit.git'
```

or

```ruby
gem "capybara", '~>2.0'
gem 'capybara-webkit'
```
